cmd/suggest.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/manifoldco/promptui"
+	"github.com/speakeasy-api/speakeasy/internal/validation"
+	"github.com/spf13/cobra"
+)
+
+var suggestCmd = &cobra.Command{
+	Use:   "suggest",
+	Short: "Validate an OpenAPI document and get fixes suggested by ChatGPT",
```

Comment:
nit/question: Are these suggestions technically being generated by ChatGPT? Or is it simply GPT3?

Reply:
GPT3.5 which I believe is a chat model